### PR TITLE
Fix #630: Add frontend for resend E-mail feature.

### LINKED ISSUE #630: Auth: Need email verification mail resend feature.
Why??
Suppose user loses verification email without verifying the link. Then neither user will verify his account nor create a new account with same email address. 

# Requirements
## Backend 
- [ ] API endpoint to resend verification email (User token based)
@deshraj : Please take care of it.

## Frontend 
I think there should be button/Link ("Resend Verification Email" ) inside rounded (dark red) portion of following screenshot. 
screenshot from 2017-02-05 11 39 21

@aka-jain : Need your thoughts about it.

### PR DESCRIPTION
**Pull request by isht3**
Fixes #630. 

Before hovering with mouse.

After hovering with mouse.

After it's clicked.

### CODE REVIEW

File: frontend/src/views/web/permission-denied.html [hunk 1]
```diff
@@ -9,6 +9,13 @@
                     <div class="card-content">
                         <p><strong>Unable to create or select a team.</strong>
                             <br> {{perm.emailError}}</p>
+                        <br>
+                        <div ng-if="!perm.sendMail">
+                            <p> <span class="text-light-black">Didn't get the email?</span> <a class="waves-effect waves-dark dark-link ev-header-overflow" ng-click="perm.requestLink()">Resend Email</a>.</p>
+                        </div>
+                        <div ng-if="perm.sendMail">
+                            <p> <span class="text-light-gray">The verification mail was sent.</span></p>
```
Comment (aka-jain) [inline frontend/src/views/web/permission-denied.html]: @isht3 Please change this message to "The verification mail has been sent to you. Please confirm!"
cc @deshraj

Comment (isht3) [reply frontend/src/views/web/permission-denied.html]: @aka-jain Done!

**Comment by spyshiv** [comment]
@RishabhJain2018 : This PR depends on PR #1355 . So please review #1355 ASAP

**Comment by RishabhJain2018** [comment]
@spyshiv Yes sure. It will be done by tonight.

**Pull request merged**